### PR TITLE
Public Cloud GCE Terraform version downgrade

### PR DESCRIPTION
**Pull request by pdostal**
Hello,

we have to upgrade our Terraform file for GCE. This is just a hotfix.

- Related ticket: [poo#91944](https://progress.opensuse.org/issues/91944)
- Verification run: [problem](https://openqa.suse.de/tests/5917530) [solution](http://pdostal-server.suse.cz/tests/11765)

### CODE REVIEW

File: tests/publiccloud/transfer_repos.pm [hunk 1]
```diff
@@ -31,7 +31,8 @@ sub run {
     if (get_var('QAM_PUBLICCLOUD_SKIP_DOWNLOAD') == 1) {
         record_info('Skip download', 'Skipping download triggered by setting (QAM_PUBLICCLOUD_SKIP_DOWNLOAD = 1)');
     } else {
-        assert_script_run("du -sh ~/repos");
+        assert_script_run('du -sch ~/repos/download.suse.de/ibs/SUSE:/Maintenance:/*');
```
Comment (asmorodskyi) [inline tests/publiccloud/transfer_repos.pm:34]: is it related to terraform profile issue ?

Comment (pdostal) [reply tests/publiccloud/transfer_repos.pm:34]: It is not. It is just a debug command.

Comment (asmorodskyi) [reply tests/publiccloud/transfer_repos.pm:34]: do you want to keep it ?

Comment (pdostal) [reply tests/publiccloud/transfer_repos.pm:34]: Should I put it away or is this OK?

Comment (pdostal) [reply tests/publiccloud/transfer_repos.pm]: It's gona. Please merge when CI is done :)

**Pull request merged**